target-modules/src/main/resources/target-versions.properties

```diff
@@ -11,6 +11,7 @@ mule-sftp-connector=1.3.7
 mule-vm-connector=${mule.vm.connector.version}
 mule-objectstore-connector=1.1.5
 mule-aggregators-module=${mule.aggregators.module.version}
+mule-salesforce-connector=10.2.0
```

Comment:
if we release 10.3.0 in time, maybe change this also?

Reply:
Yes, I think this should be updated with every Salesforce release, but for now, we could leave it like this because what we have until now works with 10.2.0